src/app/common/router/navigateAfter.js

```diff
@@ -23,11 +24,13 @@ export default function navigateAfter(name, actionType, state) {
 
 export function navigateAfterEpic(action$, store, { API }) {
   return action$.ofType(navigateAssync)
-    .switchMap(function({ payload }) {
+    .switchMap(function({ payload, meta, type }) {
       const { actionType, path, state } = payload
-      return action$.ofType(actionType)
+      return action$.filter(action => {
+        return action.type === actionType && (!get(state, 'namespace') || get(action, 'meta.resource.namespace') === get(state, 'namespace'))
+      })
```

Comment:
navigateAfter should navigate after action. it should not known about other application parts including resource